cli/src/main/kotlin/commands/EvaluatorCommand.kt

```diff
@@ -179,8 +179,8 @@ class EvaluatorCommand : OrtCommand(
 
     private val repositoryConfigurationFile by option(
         "--repository-configuration-file",
-        help = "A file containing the repository configuration. If set, the '$ORT_REPO_CONFIG_FILENAME' overrides " +
-                "the repository configuration contained in the ORT result from the input file."
+        help = "A file containing the repository configuration. If set, the '$ORT_REPO_CONFIG_FILENAME' file from " +
```

Comment:
- The new description to me sounds as if the evaluator would read the ort.yml from the repository if the option is not provided, which would be misleading. 
- The repository configuration contained in the ORT file does not necessarily original from ORT_REPO_CONFIG_FILENAME.
-  nit: Also if $ORT_REPO_CONFIG_FILENAME is provided as argument for that option, that file is not ignored.

Would it make sense to say:  "... If set, the repository configuration contained in the ORT file is overridden by the given one."